src/FsCheck/Runner.fs

```diff
@@ -120,6 +122,42 @@ module Runner =
                     yield EndShrink result
               yield! test newSize resize rnd1 gen
         }
+        
+#if PCL // sorry windows phone no parallelism for you
+    let private parallelTest limit = test
+#else
+    let internal parallelTest limit initSize resize rnd0 gen =
+        (initSize, rnd0)
+        |> Seq.unfold (fun (initSize, rnd) ->
+            let rnd1, rnd2 = Random.split rnd
```

Comment:
If I understand you properly then you are about to rewrite sequential `test` like this:

```fsharp
   let rec private test initSize resize rnd0 gen =
        seq { let rnd1 = rnd0.Next ()
              let newSize = resize initSize
              let result, shrinks =
                  try
                    let (MkRose (Lazy result,shrinks)) = Gen.eval (newSize |> round |> int) rnd1 gen
                    result, shrinks
                  with :? DiscardException -> 
                    Res.rejected, Seq.empty

              // yield TestStep / shrinking
              yield! test newSize resize rnd1 gen
        }
```

Now same `rnd1` is used for `Gen.eval` and param to next call of `test`

My concern is to keep results of parallel execution consistent with results of sequential algorithm. Otherwise user will have hard times if he first found falsifiable seed in _par_ and then switching to _seq_ with same seed for debugging.